docs/tech/attributes.md

```diff
@@ -5,7 +5,7 @@ Attributes values are encrypted using the keys defined below (section Attributes
 
 ## Attributes Keys
 
-The attributes are encrypted using a key derived from a hierarchy called PreKey tree. This approach consists in deriving keys from a parent key, allowing the parent key owner, get access to derived keys. This way keys used to encrypt attributes are different among attributes and TCerts avoiding linkability while allowing an authorized auditor who owns a parent key to derive the keys in the lower levels.  
+The attributes are encrypted using a key derived from a hierarchy called PreKey tree. This approach consists in deriving keys from a parent key, allowing the parent key owner, get access to derived keys. This way keys used to encrypt attributes are different among attributes and TCerts avoiding linkability while allowing an authorized auditor who owns a parent key to derive the keys in the lower levels.
```

Comment:
Attributes are encrypted using a key derived from a key hierarchy, denoted as the `PreKey tree`. 

The `Pre-Key tree` key hierarchy provides a powerful pattern for generating and using cryptographic keys, in which keys at a lower level of the tree are derived from their parent/master key. In general, such key hierarchy allows one to use different keys for different data.

In this context, the `PreKey tree` provides us with the ability to use different keys to encrypt different attributes (even within the same `TCert`). This is useful both for addressing potential transaction linkability concerns and for simplifying key management.

Note that the `PreKey tree` allows an authorized auditor (who owns a parent/master key) to derive the keys further down in the key hierarchy without requiring further communication with other entities.